src/segment-loader.js

```diff
@@ -1154,6 +1154,14 @@ export default class SegmentLoader extends videojs.EventTarget {
       end = this.duration_();
     }
 
+    // skip removes that would throw an error
+    // commonly happens during a rendition switch at the start of a video
+    // from start 0 to end 0
+    if (end <= start) {
```

Comment:
I was seeing a sourcebuffer remove error being thrown when trying to remove from 0 -> 0 at the start of a video, as end is <= start.